app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/index.js

```diff
@@ -117,7 +117,7 @@ export default function PipelineDetailsButtons({
   stopError,
   change,
 }) {
-  const isLatestVersion = change ? change.latest === 'true' : true;
+  const isLatestVersion = change ? change.latest === true : true;
```

Comment:
i think `change ? change.latest : true` is fine

Reply:
sounds good, updated